.editorconfig

```diff
@@ -0,0 +1,23 @@
+root = true
+
+[*]
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+
+[*.{sh}]
+end_of_line = lf
+indent_size = 2
+tab_width = 2
+
+[*.{ps1}]
+indent_size = 2
+tab_width = 2
+
+[*.{cs}]
+indent_size = 4
+tab_width = 4
+
+[*.{*proj,xml,props,targets,tasks}]
+indent_size = 2
+tab_width = 2

```

Comment:
Can/should we use the same .editorconfig as is in dotnet/runtime?

Reply:
I copied it from Arcade. We can use the runtime one.